dom/src/test/java/org/apache/james/mime4j/field/LenientDateTimeFieldTest.java

```diff
@@ -44,6 +45,19 @@ public void testDateDST() throws Exception {
         Assert.assertEquals(new Date(1216221153000L), f.getDate());
     }
 
+    @Test
+    public void extraPDTShouldBeTolerated() throws Exception {
+        DateTimeField f = parse("Date: Wed, 16 Jul 2008 17:12:33 +0200 (PDT)");
+        Assert.assertEquals(new Date(1216221153000L), f.getDate());
+    }
+
+    @Ignore("java.time.format.DateTimeParseException: Text 'Thu, 4 Oct 2001 20:12:26 -0700 (PDT),Thu, 4 Oct 2001 20:12:26 -0...' could not be parsed at index 36")
```

Comment:
Desperate on this.

I'm trying to find a way to tell DateTimeFormatter: and now ignore extra input, just like SimpleDateFormat is doing....

https://stackoverflow.com/questions/68016367/migrating-from-simpledateformat-to-datetimeformater-extra-input

Some idea maybe @mbaechler ?

Reply:
did you really had a stackoverflow answer in, like, 15 minutes?

Reply:
No I found by myself, like a great boy...

Tried another approach: if I can't get the formatter to do what I want then maybe it's about the way I use it.